core/src/main/java/com/linecorp/armeria/server/Server.java

```diff
@@ -171,11 +170,12 @@ public String defaultHostname() {
      * Returns the primary {@link ServerPort} that this {@link Server} is listening to. This method is useful
      * when a {@link Server} listens to only one {@link ServerPort}.
      *
-     * @return {@link Optional#empty()} if this {@link Server} did not start
+     * @return {@code null} if this {@link Server} did not start
```

Comment:
Feels a bit odd to have two `Return` / `@return` in a single javadoc.

Reply:
Seems to me that the Javadoc body gets too long if I move what's written in `@return`..